features/step_definitions/table_steps.rb

```diff
@@ -69,3 +99,11 @@
     step lstep
   end
 end
+
+Then "(they )should see the following table" do |table|
+  assert has_table?(rows: table.raw)
+end
+
+Then "the table should be ordered by {string}" do |column|
```

Comment:
We have a step `When "table is sorted by {string}" do |column|`, maybe be consistent and use "sort" or "order" in both places.

Reply:
`table is sorted` makes the table sorted (clicks on the column header)
`table should be sorted` asserts the table is currently sorted (only reads table)